Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -42,6 +38,22 @@ fileprivate extension SPSettingsViewController {
     }
 }
 
+extension SPSettingsViewController {
+    @objc
+    var isActiveSustainer: Bool {
+        SPAppDelegate.shared().simperium.preferencesObject().isActiveSubscriber
+    }
+
+    @objc
+    func sustainerSwitchDidChangeValue(sender: UISwitch) {
+        let isOn = sender.isOn
+
+        let iconName = isOn ? SPSustainerAppIconName : nil
+        UserDefaults.standard.set(isOn, forKey: .useSustainerIcon)
```

Comment:
Q: do you think we'd wanna sync this? (If so, it could go in the `Preferences` NSMO